### DISCUSSION TITLE
Symfony.com account with darkmode not readabble (text is dark)

### DISCUSSION THREAD

**Discussion by ivanbogomoloff** [symfony/symfony#49165] [Q&A] [answered]
Hello. I signed to account in chrome with force-darkmode and saw that text is dark.
![image](https://user-images.githubusercontent.com/2998109/215701751-e5de82b9-8592-4bec-8e5b-a1572f1ff871.png)

Liitle thing makes me sad

**Comment by javiereguiluz** [comment, marked as answer]
Thanks a lot for reporting this issue. It's just been fixed. Thanks!
